base/libgit2/callbacks.jl

```diff
@@ -224,7 +229,7 @@ function credentials_callback(libgit2credptr::Ptr{Ptr{Void}}, url_ptr::Cstring,
 
     if isset(allowed_types, Cuint(Consts.CREDTYPE_USERPASS_PLAINTEXT))
         defaultcreds = reset!(UserPasswordCredentials(true), -1)
-        credid = "$schema$host"
+        credid = "$(isempty(schema) ? "ssh" : schema)://$host"
```

Comment:
Slight change to behaviour here. It used to be that when the `schema` was not defined in the URL the credential ID would be `"$host"` now it will be `"ssh://$host"`.